### ISSUE TITLE
As stakeholder, I want to have the Authorizations API exposed on OpenZaak

### ISSUE THREAD

**Issue by joeribekker** [open-zaak/open-zaak#23]
... so applications can request their permissions (and theoretically, set permissions).

**Description**
The Authorization API was left out of scope in #3. Which might or might not be the best choice, depending on #3. This US makes sure it gets in.

**Comment by sergei-maertens** [comment]
PR for AC was merged recently.
